l3xz-leg-ctrl-firmware.ino

```diff
@@ -283,6 +283,26 @@ void setup()
   /* Initialize MCP2515 */
   mcp2515.begin();
   mcp2515.setBitRate(CanBitRate::BR_250kBPS_16MHZ);
+
+  CanardFilter const CAN_FILTER_SERVICES = canardMakeFilterForServices(node_id);
+  DBG_INFO("CAN Filter #1\n\r\tExt. Mask : %8X\n\r\tExt. ID   : %8X",
+           CAN_FILTER_SERVICES.extended_mask,
+           CAN_FILTER_SERVICES.extended_can_id);
+
+  /* Only pass service requests/responses for this node ID through to receive buffer #0. */
+  uint32_t const RXMB0_MASK = CAN_FILTER_SERVICES.extended_mask;
```

Comment:
Are you sure you're not supposed to set `MCP2515::CAN_EFF_BITMASK` here as well?

Reply:
Can you `candump` the bus, find your service request there, and ensure that its ID matches the filter?

Reply:
I found a bug in the MCP2515 driver, fixed in https://github.com/107-systems/107-Arduino-MCP2515/pull/86.